### DISCUSSION TITLE
Traffic getting out from tun0 but not back in

### DISCUSSION THREAD

**Discussion by xrpdevs** [pacien/tincapp#116] [Q&A]
Using a Honor p20 pro (android 11)

IP and route look fine, I have tons of other devices on my tinc network to compare against.

If I ping, or try other protocols from the phone, eg, wget I can see the outbound packets coming in via tcpdump on the remote, but no traffic ever gets back to the phone.

**Comment by xrpdevs** [comment]
The link itself is established, as I can see routes etc in the app, and all other nodes that should be reachable have the reachable status

I don't have root on this device as its my daily driver, so I can't poke around with iptables etc.

**Comment by pacien** [comment]
Quoting Jamie Prince (2022-12-14 12:19:12)
> If I ping, or try other protocols from the phone, eg, wget I can see
> the outbound packets coming in via tcpdump on the remote, but no
> traffic ever gets back to the phone.

Do you see some traffic activity on the app's log screen on the phone
while trying to reach it out from elsewhere?
